Gaijinizer patch/Map045.txt

```diff
@@ -104,22 +105,23 @@ There has to be some reason...#Aesthetic
 何と言われようと、必ず助け出してみせる…！
 > CONTEXT: Map045/events/4/pages/0/60/Dialogue/kazuya
 > CONTEXT: Map183/events/4/pages/0/60/Dialogue/kazuya
-Besides, the fact of the matter is, she is being chased
-by M.O.W. No matter what she says, we have to help her.#Aesthetic
+Besides, the fact of the matter is, she is being
```

Comment:
This isn't necessarily your work, but I feel that we might want to change it to say:

> **_Besides, the fact of the matter is that she's being
 chased by M.O.W._** No matter what she says, we have
to help her.

Reply:
Yeah sounds good. I think I just fixed the line overflow on that one.

By the way, it's really petty but having M.O.W. with all those old-style acronym periods has been bugging me. Do you think it'd work better without them? e.g. "she's being chased by MOW."